validator/sawtooth_validator/journal/journal.py

```diff
@@ -70,22 +70,34 @@ def stop(self):
             self._exit = True
 
     class _PublisherThread(Thread):
+        CHECK_PUBLISH_BLOCK_FREQUENCY = 0.1
```

Comment:
recommend passing this in as a keyword argument to __init__ with a default. Pass it in from the top level validator class down to whatever instantiates _PublisherThread so that we can establish it via a config file in the future.